lib/reflectAll.js

```diff
@@ -1,5 +1,7 @@
 import reflect from './reflect';
 import isArray from 'lodash/isArray';
+import map from 'lodash/map';
```

Comment:
@megawac I can't replace this function because lodash v4.8.0 arrayMap doesn't exists.